Tests/UnitTests/Networking/Backend/BaseBackendPostReceiptDataTestClass.swift

```diff
@@ -17,10 +17,15 @@ import XCTest
 
 @testable import RevenueCat
 
-class BackendPostReceiptDataTests: BaseBackendTests {
+class BaseBackendPostReceiptDataTestClass: BaseBackendTests {
 
-    override func createClient() -> MockHTTPClient {
-        super.createClient(#file)
+    override func invokeTest() {
```

Comment:
This seems weird. Does it make Xcode show all of these tests duplicated and they pass without doing anything?

Reply:
Yeah, it's the easiest way I found to get the tests to only for a given platform without littering each test func in each platform-specific class with a bunch of availability checks. Definitely might be a better way, though.

Reply:
Did you consider something similar to what I did for integration tests to split between SK1 and SK2? You could make the post iOS 13 tests the base one, `XCTSkip` them for 12, then have the separate tests for iOS 12.

Reply:
Oooo I didn't see that, I'll take a look. That sounds a lot better.

Reply:
The problem with that solution is that I want to run the exact same tests, just different test data. With the SK1/SK2 differences, from what I understand it's different APIs.

Reply:
Yeah it's a slightly different example, but you could make one of the classes override the other without needing this "abstract" one with tests that don't actually run?

Reply:
I tried it, but it ends up using the same test data files since for some reason they all point to the original `base` class that is overridden as the destination for the json. I was thinking about implementing my own:
`public func assertSnapshot<Value, Format>(`
so that I could pass in:
`snapshotDirectory: String? = nil,` to  `public func verifySnapshot<Value, Format>(`
instead of taking their default value for `snapshotDirectory: String? = nil`

Reply:
Thursday Josh will look at this again.

Reply:
Yeah, my most recent update is much better, @NachoSoto